README.md

```diff
@@ -57,27 +57,34 @@ The `grace` workflow consists of the following steps:
 
 `grace` has been tested with Python 3.8+ on OS X.
 
-For local development, clone the repo and install in editable mode:
+For local development, clone the repo and install in editable mode following these guidelines:
+
+*Note:* Choose which conda environment you'd like to use:
+* If you need to annotate your data (images) in *napari*, we recommend the `grace-env-with-napari` -> `environment-with-napari.yaml`
+* If you rely on previously annotated data and do not require *napari*, we recommend the `grace-env-napari-free` -> `environment-napari-free.yaml`
```

Comment:
Having napari in both names might be confusing (or confused with a napari "free" version as opposed to paid?). Maybe something like environment-without-annotation or something along those lines

Reply:
did anything change related to this comment?